features/breol_news/breol_news.install

```diff
@@ -8,11 +8,42 @@
 /**
  * Move news content from body to lead.
  */
-function breol_news_update_7101() {
-  $nodes = node_load_multiple([], ['type' => 'breol_news']);
-  foreach ($nodes as $node) {
-    $node->field_lead = $node->body;
-    $node->body = [];
-    node_save($node);
+function breol_news_update_7101(&$sandbox) {
+  $items_per_batch = 50;
+
+  // Initial setup.
+  if (!isset($sandbox['processed'])) {
+    // Load the nids to update.
+    $nids = db_query('SELECT nid FROM {node} WHERE type = :type', ['type' => 'breol_news'])->fetchCol();
```

Comment:
Maybe an where to limit it to nodes already moved (ala `field_body != ‘’`) so if the database dies the process can be continued.